script/test

```diff
@@ -86,6 +86,9 @@ readonly FULL_LOGS
 TREL=${TREL:-0}
 readonly TREL
 
+MULTIPAN_RCP="${MULTIPAN_RCP:-1}"
```

Comment:
Does it make sense to default this option to 1, or is this option even needed in the script at all? This will effect all of the workflow options that run the test script to build the rcp simulation/posix libraries.

Another option could be to add in a new stage, maybe in the posix.yml workflow, to run one stage of tests with OT_MULTIPAN_RCP=ON and the OT_BUILDDIR=${PWD}/build/multipan (or something like that?). 

I think that this would simplify the amount of changes that need to be made for these tests to be run in the workflow and minimize the impact on other stages. However, would this increase the complexity needed to access the apps in the expect test you wrote?

Reply:
Make sense to disable it by default. Just pushed the changes.